src/documents/IoCTestingFramework.md

```diff
@@ -35,57 +39,68 @@ lifecycle of the component holding the listener. This is in order to make applic
+Writing fixtures using the IoC Testing framework requires the test implementor to derive from two special [grades](ComponentGrades.md), `fluid.test.testEnvironment` and `fluid.test.testCaseHolder`, 
+which are packaged within the testing framework implementation in the
 file `IoCTestUtils.js`. The tester must derive their own component types from these grades, and assemble
 them into various component trees corresponding to the desired integration scenarios.
 
-The first type of component corresponds to the overall root of the component tree under test - the test *environment*, defined in the grade
-`fluid.test.testEnvironment`. The children of this component correspond to the entire "application segment" (the context) under test -
+The first type of component corresponds to the overall root of the component tree under test - the *test environment*, defined in the grade
+`fluid.test.testEnvironment`. The children of this component correspond to the entire "application segment" under test -
 this may be as large (as an entire application) or as small (as a single component) as required in order to comprise the desired fixture.
 These children are intermixed with components of the second type, the *test fixtures*, derived from the grade `fluid.test.testCaseHolder`.
-These fixture components typically contain no implementation code, but are simply holders for declarative JSON configuration defining the
+These fixture components are holders for declarative JSON configuration defining the
 sequence and structure of a group of test cases which are to be run.
 
-## Simple Example ##
+### `modules`, `tests` and `sequence` ###
 
-This simple example shows the testing of a simple component, `fluid.tests.cat` which defines one method. Firstly we define the component under test:
+The standard structure inside a `fluid.test.testCaseHolder` has an outer layer of containment, `modules`, with members corresponding to
+QUnit [modules](https://api.qunitjs.com/QUnit.module/), and within that an entry named `tests`, holding an array of structures corresponding to QUnit [test](https://api.qunitjs.com/QUnit.test/). In 
+ordinary use, each element `tests` then contains a member named `sequence` holding a list of [*fixture records*](#supported-fixture-records).
+
+As well as containing a flat list of fixture records, `sequence` may also contain nested arrays of such records. These nested arrays will be
+flattened into a single array by use of the utility [`fluid.flatten`](CoreAPI.md#fluid-flatten-array-) before being processed. This helps in
+assembling complex sequences out of previously canned sequence segments. However, building up complex, reusable test sequences is best done by
+use of the [`sequenceGrade`](/IoCTestingFramework.md#using-sequencegrade-to-build-up-complex-reusable-test-sequences) element, instead of the `sequence` element.
```

Comment:
The "sequenceGrade" link returns a 404 page when following the link on GitHub. It may have to do with the leading "/".